### PR TITLE
Improved json format of data being passed from download api

### PR DESCRIPTION
**Pull request by samiyaakhtar**
Output now follows format:

```
[
  {
    "image_height": 364,
    "image_id": 111,
    "image_width": 580,
    "imagelocation": "https://mock-storage.blob.core.windows.net/new-uploads/ZRK6.jpg",
    "labels": [
      {
        "classificationname": "rattlesnake",
        "x_max": 182.62,
        "x_min": 99.78,
        "y_max": 50.26,
        "y_min": 50.26
      },
      {...}
```

### CODE REVIEW

**Comment by mtarng** [comment]
@samiyaakhtar does this handle the cases where a single label can have multiple classifications?

**Comment by samiyaakhtar** [comment]
@mtarng yeah this should just pull labels as an array, will test the scenario to make sure

File: utils/vott_parser.py [hunk 1]
```diff
@@ -27,8 +19,18 @@ def __build_tag_from_VottImageTag(image_tag):
         "y2": image_tag.y_max,
         "width": image_tag.image_width,
         "height": image_tag.image_height,
-        "tags": image_tag.classification_names,
-        "location": image_tag.image_location
+        "tags": [image_tag.classification_names],
+        "location": image_tag.image_location,
```
Comment (andrebriggs) [inline utils/vott_parser.py]: @samiyaakhtar is this image location going to be the location on the local disk? Also is "location" necessary? This is an example of json I see produced from VOTT

```
 "4.JPG": [
            {
                "x1": 242.69947363429438,
                "y1": 111.94245383522727,
                "x2": 386.5604846358118,
                "y2": 288.6293767755682,
                "width": 724,
                "height": 483,
                "box": {
                    "x1": 242.69947363429438,
                    "y1": 111.94245383522727,
                    "x2": 386.5604846358118,
                    "y2": 288.6293767755682
                },
                "UID": "9022c82d",
                "id": 3,
                "type": "Rectangle",
                "tags": [
                    "land"
                ],
                "name": 1
            }
        ],
```

Comment (samiyaakhtar) [reply utils/vott_parser.py]: Location may not be necessary to load it but the other fields are, however when i give it a url location it loads the labels successfully as long as they have the other mandatory fields

Comment (andrebriggs) [reply utils/vott_parser.py]: @samiyaakhtar let's remove it then.

Comment (samiyaakhtar) [reply utils/vott_parser.py]: removed

**Review by andrebriggs — APPROVED** [review]

**Pull request merged**